textplot/utils.py

```diff
@@ -3,13 +3,83 @@
 import re
 import numpy as np
 import functools
+import copy
 
 from collections import OrderedDict
 from nltk.stem import PorterStemmer
 from itertools import islice
 
+import spacy
+from nltk.stem.snowball import SnowballStemmer
 
-def tokenize(text):
+class SpacyTokenizer(object):
+    def __init__(self, lang='de', lower=False):
+
+        if lang == 'de':
+            spacy_model = 'de_core_news_sm'
```

Comment:
adding some logic to download the minimal model for a language if not installed would be nice here :)

Reply:
perhaps also making the default `lang='en'` would make more sense too.